src/fosslight_dependency/_package_manager.py

```
@@ -130,7 +171,7 @@ def run_gradle_task(self):
                 if ret_plugin:
                     cmd = f"{cmd_gradle} generateLicenseTxt"
                     try:
-                        ret = subprocess.check_output(cmd, shell=True, encoding='utf-8')
+                        ret = check_output_safe(cmd, shell=True)
```

Ruff (0.15.7)

[error] Function call with `shell=True` parameter identified, security issue

(S604)